waiter/resources/web/authenticated-redirect.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+    <title>Working...</title>
```

Comment:
Can we use a title that better captures what this page is doing?

Reply:
what do you think is better

Reply:
`Completing SAML Authentication...`